client-v2/src/components/transitions/FadeTransition.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { CSSTransition } from 'react-transition-group';
+import { styled } from 'constants/theme';
+
+type TransitionDirections = 'up' | 'right' | 'down' | 'left';
+
```

Comment:
A handy container for fade transitions. Hopefully it's useful to others outside of the context of this PR.